### PR TITLE
Fixing logs autoscroll behavior

### LINKED ISSUE #1709: Log autoscroll is not working v4.0.0
**Describe the bug**
Older versions log screen was opening popup window and autoscroll newest log by default. In v4.0.0 log screen opening like terminal window and search box added. This become more usefull but no autoscroll. Please add this feature default and user can stop by clicking icon on top bar. 

image

### PR DESCRIPTION
**Pull request by aleksfront**
Fixes #1709 

working autoscroll

Signed-off-by: Alex Andreev <alex.andreev.email@gmail.com>

### CODE REVIEW

File: src/renderer/components/dock/pod-log-list.tsx [hunk 1]
```diff
@@ -95,15 +95,9 @@ export class PodLogList extends React.Component<Props> {
...
+    const { scrollOffset } = props;
 
-    if (scrollDirection == "backward") {
-      this.isLastLineVisible = false;
-    } else {
-      if (clientHeight + scrollOffset === scrollHeight) {
-        this.isLastLineVisible = true;
-      }
-    }
+    this.isLastLineVisible = clientHeight + scrollOffset === scrollHeight;
```
Comment (Nokel81) [inline src/renderer/components/dock/pod-log-list.tsx]:
```suggestion
    this.isLastLineVisible = (clientHeight + scrollOffset) === scrollHeight;
```

This is confusing to read at first glance.

**Review by Nokel81 — APPROVED** [review]

**Review by ixrock — APPROVED** [review]

**Comment by hodolomax** [comment]
Will you add stop/start function for autoscroll?

**Comment by aleksfront** [comment]
@hodolomax Autoscroll function is active when last log line is visible. If scrolled up a bit, this feature will be switched off.

**Comment by hodolomax** [comment]
@aleksfront great! thank you

**Comment by Nuru** [comment]
Still broken in 4.0.3

**Comment by Nokel81** [comment]
@Nuru can you provide some more information please? A screenshot of your version information and a gif of the logs screen would be excellent.

**Comment by Nuru** [comment]
@Nokel81 See #1791

**Pull request merged**